app/models/legal_aid_application.rb

```diff
@@ -446,6 +446,34 @@ def family_linked_associated_application?
     lead_linked_application&.link_type_code == "FC_LEAD"
   end
 
+  def family_linked?
+    @family_linked ||= associated_family_linked_application? || lead_family_linked_application?
+  end
+
+  def legal_linked?
+    @legal_linked ||= associated_legal_linked_application? || lead_legal_linked_application?
+  end
+
+  def family_linked_lead_or_associated
+    return nil unless family_linked?
+
+    lead_family_linked_application? ? "Lead" : "Associated"
+  end
+
+  def legal_linked_lead_or_associated
+    return nil unless legal_linked?
+
+    lead_legal_linked_application? ? "Lead" : "Associated"
+  end
+
+  def family_linked_applications_count
+    family_linked_count.positive? ? family_linked_count + 1 : nil
```

Comment:
You could use your `lead_family_linked_application?` method if you wanted here and the same for legal link method below

Reply:
👍 thanks @agoldstone93 amended